dotnet/README.md

```diff
@@ -0,0 +1,16 @@
+# Tool builder: gcr.io/cloud-builders/dotnet
+This directory contains the description for the `dotnet` Container Build step. This step will run the .NET Core `dotnet` command line. This builder has two SDKs installed:
++ 1.0.3 with SDK Preview 2 build 3156, to support .NET Core 1.0.5 with project.json solutions.
++ 1.0.4, which supports .csproj projects for .NET Core 1.0.5 and 1.1.2.
+
+Any arguments passed to this builder step will be passed directly to the `dotnet` tool. Note that when using the `dotnet restore` command you should always specify the `--packages` option to a path within the `/workspace`, this way the restored packages will be available to the next invocations of `dotnet`. See the [example](examples/TestApp/cloudbuild.yaml) for how use `dotnet resotre` correctly.
```

Comment:
It does and in the cloudbuild.yaml file it uses a relative path. In this document I mention /workspace to explain why we need to do that. Perhaps I should reword the doc.